src/components/VersionNumber.js

```diff
@@ -19,7 +19,7 @@ export const VersionNumber = () => {
     </Wrapper>
   );
 };
-
+//App-Version can be hardcoded ?
```

Comment:
I am not sure what this is supposed to tell me.

Reply:
that was a comment for myself that i forgot to remove . there to decide if App-Version should go to consts as well, but for me the answer is no , it will be non sense repetition for me , if I do: 
appVersion: 'App-Version'

Reply:
solved in 71c498a

Reply:
i think it is weird to have a colon in the end of the string, if it is already bracketed. can you change it to use brackets only? (there is another place where you do that as well, please check that as well)

Reply:
yes you are right ! doen in ef58ac7be4e00799d2a955770af47c10df993eed